packages/api/lambdas/bulk-operation.js

```diff
@@ -72,14 +72,16 @@ async function applyWorkflowToGranules({
  * @param {string} [payload.index] - Optional parameter of ES index to query.
  * Must exist if payload.query exists.
  * @param {Object} [payload.ids] - Optional list of granule IDs to bulk operate on
- * @param {Function} [unpublishGranuleFunc] - Optional function to delete the
- * granule from CMR. Useful for testing.
+ * @param {Function} [removeGranuleFromCmrFunction] - passed in function used for test mocking
  * @returns {Promise}
  */
 async function bulkGranuleDelete(
   payload,
-  unpublishGranuleFunc = unpublishGranule
+  removeGranuleFromCmrFunction
 ) {
+  const concurrency = payload.concurrency || 10;
+  process.env.dbMaxPool = payload.maxDbConnections || concurrency;
```

Comment:
```suggestion
  process.env['dbMaxPool'] = payload.maxDbConnections || concurrency;
```

At some point we're going to enable the [noPropertyAccessFromIndexSignature](https://www.typescriptlang.org/tsconfig#noPropertyAccessFromIndexSignature) option, which will require this change. May as well make it now.

Reply:
I added this, however it directly conflicts with our current lint .js configuration -  https://eslint.org/docs/latest/rules/dot-notation - we can have a conversation re: this/moving forward, but I think the resolution likely falls outside the ticket scope.

Reply:
Yeah, that conflict is resolved in this PR - https://github.com/nasa/cumulus/pull/3239